mmv1/templates/terraform/update_encoder/vertex_ai_model.go.tmpl

```diff
@@ -0,0 +1,4 @@
+obj["displayName"] = obj["modelName"]
+delete(obj, "modelName")
+delete(obj, "sourceModel")
+return obj, nil
```

Comment:
This update_encoder is necessary due to source_model being included in PATCH. The API Reference doesn't include any mention of source_model for patching existing models (such as for updating the description for a copied model)

Reference: https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.models/patch